packages/react-storage/src/components/StorageBrowser/Views/Controls/AddFiles.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+import { StorageBrowserElements } from '../../context/elements';
+import { CLASS_BASE } from '../constants';
+import { FileInput } from './FileInput';
+
+const { Button } = StorageBrowserElements;
+
+const BLOCK_NAME = `${CLASS_BASE}__add-files`;
+
+export interface AddFilesControl {
+  (props: { disable: boolean }): React.JSX.Element;
+}
+
+export const AddFilesControl: AddFilesControl = ({ disable }) => {
+  const fileUploadRef = React.useRef<HTMLInputElement>(null);
+
+  return (
+    <>
+      <FileInput actionType="UPLOAD_FILES" ref={fileUploadRef} />
+      <Button
```

Comment:
Can we add variants 'add-files' and 'add-folder' to this and the AddFolder control for Buttons? That way we have a hook for theming them for now.

Reply:
thanks @hbuchel, just added them. Please let me know if there's anything else I need to add for them